api/client/network.go

```diff
@@ -222,7 +224,7 @@ func consolidateIpam(subnets, ranges, gateways []string, auxaddrs map[string]str
 				return nil, fmt.Errorf("multiple overlapping subnet configuration is not supported")
 			}
 		}
-		iData[s] = &network.IPAMConfig{Subnet: s, AuxAddress: map[string]string{}}
+		iData[s] = &network.IPAMConfig{Subnet: s, AuxAddress: map[string]string{}, Options: options}
```

Comment:
I dont completely to the fact that this is infact a subnet specific configuration. Even from a functionality standpoint, it is a network configuration specific to ipam-driver and certainly not subnet specific.

The fact that libnetwork uses ipam per subnet should not dictate the UX or the functionality.